daemon/archive_tarcopyoptions_unix.go

```diff
@@ -18,8 +18,10 @@ func (daemon *Daemon) tarCopyOptions(container *container.Container, noOverwrite
 		return nil, err
 	}
 
+	identity := idtools.Identity{UID: user.Uid, GID: user.Gid}
```

Comment:
Wondering why you added the intermediate variable here, instead of putting it inline (as it was before)